scripts/poll_spi_result

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Wait for the results of an SPI execution.
+# Receives the image_unique_id of the execution.
+# Exits with:
+# - 0 if the test passed.
+# - 1 if the test failed.
+# - 2 if it timed out checking for results.
+
+readonly TIMEOUT=7200  # 2 hours.
```

Comment:
I'm not sure of a better way to do it at the moment, and this should work fine for now, but clearly there's a chance that the job could be queued, delayed, or still running and pass just fine, but fail in the jenkins job because the result doesn't show up within the timeout.